openedx/core/apidocs.py

```diff
@@ -1,4 +1,4 @@
-"""  # lint-amnesty, pylint: disable=django-not-configured
+"""  # lint-amnesty, pylint: disable=django-not-configured  # lint-amnesty, pylint: disable=django-not-configured
```

Comment:
why it is adding lint-amnesty twice?

Reply:
Looks like some other PR also ran amnesty on this file before. Thanks for pointing it out.